docs/source/advanced/http-engine.mdx

```diff
@@ -59,25 +60,29 @@ class MyHttpEngine(val wrappedClient: MyClient) : HttpEngine {
 }
 ```
 
-This example shows an asynchronous wrappedClient that runs the network request from a separate thread but since `HttpEngine.execute` is called from a background thread itself, it is also ok to block in `execute()`.
+This example uses an asynchronous `wrappedClient` that runs the network request in a separate thread. Note that because `HttpEngine.execute` itself is called from a background thread, you can safely block in `execute()`.
 
-## Use your HttpEngine
+## Using your `HttpEngine`
 
-Once you have implemented your HttpEngine, you can register it in your [ApolloClient](https://apollographql.github.io/apollo-android/kdoc/apollo-runtime/com.apollographql.apollo3/-apollo-client/index.html) using [HttpNetworkTransport](https://apollographql.github.io/apollo-android/kdoc/apollo-runtime/com.apollographql.apollo3.network.http/-http-network-transport/index.html):
+After you create your `HttpEngine` implementation, you can register it with your [`ApolloClient`](https://apollographql.github.io/apollo-android/kdoc/apollo-runtime/com.apollographql.apollo3/-apollo-client/index.html) instance using [`HttpNetworkTransport`](https://apollographql.github.io/apollo-android/kdoc/apollo-runtime/com.apollographql.apollo3.network.http/-http-network-transport/index.html):
 
 ```kotlin
 // Use your HttpEngine
 val client = ApolloClient.Builder()
-    .networkTransport(
-        HttpNetworkTransport(
-            serverUrl = "https://com.example/graphql",
-            engine = MyHttpEngine(wrappedClient)
-        )
+  .networkTransport(
+    HttpNetworkTransport(
+      serverUrl = "https://com.example/graphql",
+      engine = MyHttpEngine(wrappedClient)
     )
-    .build()
+  )
+  .build()
 ```
 
-This is everything needed. Your GraphQL queries will then use `MyHttpEngine` to execute the HTTP requests.
+With this configuration, Apollo Android sends all of its GraphQL operation requests with `MyHttpEngine`.
 
+## Other HTTP customizations
 
+Besides implementing `HttpEngine`, Apollo Android also supports other methods for customizing HTTP behavior:
 
+* [No runtime](no-runtime): You can opt out of the Apollo Android runtime completely and only use generated models and parsers. Use this option if you don't need any of the runtime features (caching, batching, automatic persisted queries, etc.).
+* [HTTP interceptors](interceptors-http): If you want to add HTTP headers and/or logging to your requests, [HTTP interceptors](interceptors-http) enable you to do this with minimal code.
```

Comment:
That might belong to an `Introduction` or `Overview` page, [like for caching](https://www.apollographql.com/docs/android/v3/caching/introduction/)?

Reply:
Agreed! I'll investigate that as a next step, but will leave it here for this PR to make sure the content isn't lost.